Source/MBF.SAMCD.GPIO.pas

```diff
@@ -96,7 +96,15 @@     TArduinoPin = record
       A4 = TNativePin.PA5;   A5 = TNativePin.PB2;
     end;
   {$endif}
-  {$if defined(SAMD20XPRO) or defined(SAMD21XPRO) }
+  {$if defined(SAMD20XPRO)}
+  type
+    TArduinoPin = record
+    const
+      None=-1;
+      D13= TNativePin.PA14; // just for blinky to work
```

Comment:
This is not the right way to do things. Better define a const LED (and LED0) with value TNativePin.PA14.
You should not 'kill' the ArduinoPins Definition for the sake of making Blinky run.

Reply:
Fully agreed.
But the current Blinky used this Arduino-pin, and the board itself has no Arduino pins.
Please correct me if I am wrong.

Reply:
Haha !
Did not know of this adapter-board !
Nice addition.
Surely, the Arduino pin-definitions must be included.